languageFiles/es.json

```diff
@@ -9,7 +9,7 @@
   "CC Options": "Opciones de subtitulado",
   "On": "Sí",
   "Off": "No",
-  "Discover": "Descubrir",
+  "Discovery": "Descubrir",
```

Comment:
Steve says this should be left as is as Discover. This is assuming this is the string at the top of the discovery UI:

![screen shot 2017-08-09 at 4 34 22 pm](https://user-images.githubusercontent.com/14931241/29148164-9dc3e910-7d20-11e7-8fea-9b49617779e4.png)